### PR TITLE
Introduce gflags as dependency for parsing command line arguments.

### PR DESCRIPTION
**Pull request by wp19991**

### CODE REVIEW

File: bazel/metable_deps_setup.bzl [hunk 1]
```diff
@@ -49,6 +49,12 @@ def metable_deps_setup():
         strip_prefix = "googletest-release-1.11.0",
     )
 
+    git_repository(
```
Comment (jovany-wang) [inline bazel/metable_deps_setup.bzl:52]: Could you use `maybe`?

Comment (wp19991) [reply bazel/metable_deps_setup.bzl:52]: ok

**Comment by jovany-wang** [comment]
Also you need use https://github.com/jovany-wang/metable/blob/main/scripts/code-format.sh script to format your codestyle.

File: bazel/thirdparty/gflags.BUILD [hunk 2]
```diff
@@ -0,0 +1,5 @@
+cc_binary(
```
Comment (jovany-wang) [inline bazel/thirdparty/gflags.BUILD:1]: Why we need  this file?

**Pull request merged**